create_dmd_release/create_dmd_release.d

```diff
@@ -371,6 +371,8 @@ void buildAll(Bits bits, string branch, bool dmdOnly=false)
     }
     auto hostDMDEnv = " HOST_DC="~hostDMD~" HOST_DMD="~hostDMD;
     auto isRelease = " ENABLE_RELEASE=1";
+    //Enable lto for everything.
+    auto ltoOption = " ENABLE_LTO=1";
```

Comment:
@maxhaton: Are you sure this actually works? I don't see `ENABLE_LTO` being forwarded to `build.d`in https://github.com/dlang/dmd/blob/master/src/posix.mak#L100.

Reply:
GNU make sets `ENABLE_LTO=1` as an environment variable which is read by `build.d`.
So `make -f posix.mak ENABLE_LTO=1` works as expected.

Reply:
Hmm okay, but what about Windows with DigitalMars make? IIRC, there's a difference, and that's why all (?) other vars are forwarded explicitly. No `ENABLE_LTO` forwarding in https://github.com/dlang/dmd/blob/master/src/win32.mak#L75 either.

Reply:
=> https://github.com/dlang/dmd/pull/13524